tests/test_preprocess.py

```diff
@@ -1,14 +1,29 @@
+from collections import OrderedDict
+
 from config import PATH_DATA, PATH_STOP_WORDS
-from preprocess import create_corpus_from_files, build_inverted_index
+from preprocess import create_corpus_from_files, build_inverted_index, InvertedIndex
 
 import pickle
+import pytest
 
-def test_build_dev_inverted_index_type_1():
-    corpus = create_corpus_from_files(PATH_DATA, dev=True, dev_iter=100)
-    index = build_inverted_index(corpus, PATH_STOP_WORDS, type_index=1)
+@pytest.mark.xfail(reason="data is not imported in the repo +  pickle doesn't load properly the data")
+def test_build_dev_inverted_index():
```

Comment:
Why not split between index types ?

Reply:
? In three different tests ? Yes should be better. But they don't work for now anyway

Reply:
ok you have it. Pytest parametrization and fixture with scope